src/status_im/ui2/screens/chat/components/chat_item.cljs

```diff
@@ -0,0 +1,199 @@
+       [badge/message-counter unviewed-messages-count])]))
+
+(defn icon-style []
+  {:color           colors/black
+   :width           15
+   :height          15
+   :container-style {:top          13 :left 72
+                     :position     :absolute
+                     :width        15
+                     :height       15
+                     :margin-right 2}})
+
+(defn chat-item-icon [muted private-group? public-group?]
+  (cond
+    muted
+    [icons/icon :main-icons/tiny-muted (assoc (icon-style) :color colors/gray)]
+    private-group?
+    [icons/icon :main-icons/tiny-group (icon-style)]
+    public-group?
+    [icons/icon :main-icons/tiny-public (icon-style)]
+    :else
+    [icons/icon :main-icons/tiny-new-contact (icon-style)]))
+
+(defn chat-item-title [chat-id muted group-chat chat-name edit?]
+  [quo2.text/text {:weight              :semi-bold
+                   :color               (when muted :secondary)
+                   :accessibility-label :chat-name-text
+                   :ellipsize-mode      :tail
+                   :number-of-lines     1
+                   :style               {:position :absolute
+                                         :left     72
+                                         :top      10
+                                         :right    (if edit? 50 90)}}
+   (if group-chat
+     (utils/truncate-str chat-name 30)
+     ;; This looks a bit odd, but I would like only to subscribe
+     ;; if it's a one-to-one. If wrapped in a component styling
+     ;; won't be applied correctly.
+     (first @(re-frame/subscribe [:contacts/contact-two-names-by-identity chat-id])))])
+
+(defn chat-list-item [home-item opts]
+  (let [{:keys [chat-id chat-name color group-chat muted emoji highlight edit? public? unviewed-messages-count last-message]} home-item
+        background-color (when highlight (colors/get-color :interactive-02))]
+    [rn/touchable-opacity (merge {:style {:height 64 :background-color background-color}} opts)
+     [:<>
+      (when (pos? unviewed-messages-count)
+        [rn/view {:position :absolute :top 2 :left 8 :right 8 :bottom 2 :border-radius 16 :background-color quo2.colors/primary-50-opa-5}])
+      [chat-icon.screen/emoji-chat-icon-view chat-id group-chat chat-name emoji
+       {:container              (assoc chat-icon.styles/container-chat-list
+                                       :top 12 :left 20 :position :absolute)
+        :size                   32
+        :chat-icon              chat-icon.styles/chat-icon-chat-list
+        :default-chat-icon      (chat-icon.styles/default-chat-icon-chat-list color)
+        :default-chat-icon-text (if (string/blank? emoji)
+                                  (chat-icon.styles/default-chat-icon-text 40)
+                                  (chat-icon.styles/emoji-chat-icon-text 40))}]
+      [chat-item-title chat-id muted group-chat chat-name edit?]
+      (when-not edit?
+        [rn/view {:height "100%" :justify-content :center}
```

Comment:
not sure why "100%"  is needed ?

Reply:
I did not write this code, its code copied from ui1